spec/api/api.yaml

```diff
@@ -914,13 +990,25 @@ components:
     ResetConfiguration:
       type: object
       properties:
-        datasets:
+        databases:
+          description: |-
+            Contains database names as keys, with values are dictionaries descrbing how the databases 
+            will be created.
           type: object
           additionalProperties:
-            type: array
-            items:
-              type: string
-          example: { 'catalog': ['db1', 'db2'] }
+            type: object
+            properties:
+              collections:
```

Comment:
You can probably use the "oneOf" keyword to specify that one or the other is used